pandas/api/tests/test_api.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pd
+from pandas import api
+from pandas.api import types
+from pandas.util import testing as tm
+
+_multiprocess_can_split_ = True
+
+
```

Comment:
@jorisvandenbossche I documented this as much as possible. Some obvious candidates for deprecation at some point.

Further this is all now tested so that anything changing the namespace will break the tests (so at least we would know/care) and take action.